db/interface.py

```diff
@@ -0,0 +1,69 @@
+# -*- coding:utf-8 -*-
+
+from db.models import Payment, Detail
+
+
+class PaymentService(object):
+    """
+    Serves as an interface which takes requests from the front-end
+    and runs the corresponding actions in the database.
+    """
+
+    def __init__(self):
```

Comment:
Upon some further testing these additions on top of my model PR, i can't seem to get the existing tests running. keep getting import errors

###Error 1###
db/__init__.py line 2 in <module>
-->from payments import app
payments.py line 9 in <module>
-->from db.interface import PaymentService
db/interface.py line 3 in <module>
-->from db.models import Payment, Detail
db/models.py line 1 in <module>
-->from db import db
__ImportError: cannot import name db__

###Error 2###
test_payments.py line 6 in <module>
-->import payments, db
payments.py line 9 in <module>
-->from db.interface import PaymentService
db/__init__.py line 2 in <module>
-->from payments import app
__ImportError: cannot import name app__

Having trouble sorting out why, likely due to the aforementioned ignorance, and it's getting a bit late so I need to review with fresh eyes tomorrow. int the interim, any thoughts?